src/stcal/jump/jump.py

```diff
@@ -343,9 +347,9 @@ def detect_jumps(
             slices.insert(
                 i,
                 (
-                    data[:, :, i * yinc : (i + 1) * yinc, :],
-                    gdq[:, :, i * yinc : (i + 1) * yinc, :].copy(),
```

Comment:
Is it OK to remove the .copy() here? I assume it's not needed?

Reply:
I would keep it in to prevent corruption of the original gdq.

Reply:
I added the copy back